sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -158,6 +158,22 @@ export function useFormatters() {
     return value ? value : DEFAULT_EMPTY_VALUE;
   };
 
+  /**
+   * Converts an empty or null string to '-', else return the actual string if the 'condition' is true.
+   * @param value string.
+   * @param condition expression
+   * @returns '-' or the actual string.
+   */
+  const conditionalEmptyStringFiller = (
+    value?: string,
```

Comment:
Should these parameters be optional?

Reply:
I think both of these parameters can be optional - `value` should be kept optional because it is getting passed into `emptyStringFiller` and there it is optional (plus depending on what it is (empty or otherwise), different values are returned. Even `condition` can be kept optional here in my opinion because if it not provided `DEFAULT_EMPTY_VALUE` will be returned, otherwise return of `emptyStringFiller` will be returned.

Reply:
if the `condition` is not there we can directly call `emptyStringFiller` from the vue right? So here does `condition` needs to be optional??